source/manual/grafana.html.md

```diff
@@ -28,6 +37,30 @@ Useful Grafana dashboards:
 - [Edge health](https://grafana.publishing.service.gov.uk/dashboard/file/edge_health.json)
 - [Application deployment dashboards](deployment-dashboards.html)
 
-The full list of Grafana dashboards is [stored in the Puppet repo][dashboards]
+The full list of Grafana dashboards is [stored in the Puppet repo][dashboards].
+For details on how to create a new dashboard, read the
+[Grafana dashboards alert documentation](/manual/alerts/grafana-dashboards.html).
 
 [dashboards]: https://github.com/alphagov/govuk-puppet/blob/master/modules/grafana/manifests/dashboards.pp
+
+## Grafana tips
+
+You can use regexes to filter for relevant information. For example,
+[`*frontend*` on the processes dashboard](https://grafana.production.govuk.digital/dashboard/file/processes.json?refresh=1m&orgId=1&var-host=All&var-processes=*frontend*&from=now-15m&to=now)
+to see all processes that have 'frontend' in them.
+
+We often show multiple metrics on the same graph. The position of the
+key shows which Y-axis each metric corresponds to:
+
+![screenshot of multiple metrics selected](/manual/images/grafana-y-axis.png)
+
+You can click on a metric in a graph to show only that metric, or you
+can `CMD + click` to select multiple:
+
+![screenshot of multiple metrics selected](/manual/images/grafana-select-multiple.png)
```

Comment:
I wonder if these will render when deployed? They're not rendering for me, but that might just be Github! https://github.com/alphagov/govuk-developer-docs/blob/2966abfb749f525761647b188163a847e0327eb8/source/manual/grafana.html.md

Reply:
I think it's just GitHub - see [the source](https://github.com/alphagov/govuk-developer-docs/blame/c54ad9c3515c3bde25cea1dada2740b99b2ff9bc/source/manual/logging.html.md#L12) and [the output here](https://docs.publishing.service.gov.uk/manual/logging.html).

Reply:
Aha! Thanks :D